site/gatsby-site/src/components/forms/QuickAddForm.js

```diff
@@ -55,7 +58,7 @@ const QuickAddForm = ({ className = '' }) => {
           message: (
             <>
               {'Report successfully added to review queue. It will appear on the  '}
```

Comment:
Is there a reason why the toast messages aren't wrapped in `t()` ?

Reply:
I forgot about them. I will create another pr for those.